torchtitan/train.py

```diff
@@ -237,18 +243,6 @@ def main(job_config: JobConfig):
         return
 
     checkpoint.load(step=job_config.checkpoint.load_step)
-    metric_logger = build_metric_logger(job_config, parallel_dims)
-
-    # plot losses loaded from checkpoint (if any) to TensorBoard
-    # NOTE: Loss info after the last log step before checkpoint saving will not be ploted.
-    #       This can be avoided by setting checkpoint.interval to be a multiple of metrics.log_freq
-    if train_state.step > 0 and not job_config.metrics.disable_logging_from_checkpoint:
```

Comment:
if you decide to remove it, then can we also remove `disable_logging_from_checkpoint`?